lib/ansible/module_utils/vmware.py

```diff
@@ -1000,6 +1000,36 @@ def get_vm_path(content, vm_name):
             folder_name = '/' + folder_name
         return folder_name
 
+    def get_vm_template(self, template_name=None):
+        """
+        Function to find the virtual machine template using name.
+        Args:
+            template_name: Name of template
+
+        Returns: virtual machine template object
```

Comment:
As this function can return VM which aren't template (if I understand correctly), you should rename it (something like "get_vm_or_template") and rewrite the docstring.